pkg/metadata/metadata.go

```diff
@@ -0,0 +1,44 @@
+package metadata
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+// metadata houses the metadata that will be written to the report directory after
+type metadata struct {
+	ClusterID      string `json:"cluster-id"`
```

Comment:
Thoughts on making a clusterMetadata Struct? 

Should then output like metadata.cluster.id and the like.

Reply:
Hrm. Not opposed to it. I'll do that.

Reply:
This seems to be not as easy as I expected, so I'm going to punt on this. It's doable, I'm just not sure of the value vs. how much workk to put into it.